package.json

```diff
@@ -16,15 +16,15 @@
     "devDependencies": {
         "@types/jest": "^29.2.3",
         "@types/lodash": "^4.14.184",
-        "@typescript-eslint/eslint-plugin": "^5.36.2",
-        "@typescript-eslint/parser": "^5.36.2",
+        "@typescript-eslint/eslint-plugin": "^5.45.0",
+        "@typescript-eslint/parser": "^5.45.0",
         "@vue/eslint-config-typescript": "^11.0.0",
```

Comment:
Apparently there’s a version 11.0.2 of this now, so we should probably update it as well?

Reply:
I left it out cause i assumed we are avoiding updating vue related packages. i guess that can't hurt

Reply:
Ah, I see… I think we can count this one as close to eslint, though.